packages/webapp/src/containers/Task/TaskLocations/index.jsx

```diff
@@ -166,7 +167,44 @@ function TaskAnimalLocations({ history, location }) {
   );
 }
 
-function TaskAllLocations({ history, location, optionalLocation }) {
+function TaskCustomLocations({ history, location }) {
+  const dispatch = useDispatch();
+  const locations = useSelector(locationsSelector);
+  const readOnlyPinCoordinates = useReadOnlyPinCoordinates();
+  const activeAndCurrentManagementPlansByLocationIds =
+    useActiveAndCurrentManagementPlanTilesByLocationIds(locations);
+  const wildManagementPlanTiles = useCurrentWildManagementPlanTiles();
+  const animalsExistOnFarm = useAnimalInventory().inventory.length > 0;
```

Comment:
NIT but since we'll also need this to hide the movement task type tile on task creation, we could consider adding this as an extra property on the `useAnimalInventory` hook result

Reply:
Yeah I was wondering if I should! 

I thought maybe since it was a simple derived value of inventory I shouldn't... but I will add it in!